config/config.toml.example

```diff
@@ -57,6 +57,9 @@ treat-old-version-utf8-as-utf8mb4 = true
 # enable-table-lock is used to control table lock feature. Default is false, indicate the table lock feature is disabled.
 enable-table-lock = false
 
+# delay-clean-table-lock is used to control delay release table lock when the abnormal situation. (Milliseconds)
```

Comment:
`is used to control delay release table lock when the abnormal situation.` -> `is used to control whether delayed-release the table lock in the abnormal situation.`